apps/examples/src/14-persistence/PersistenceExample.tsx

```diff
@@ -1,37 +1,21 @@
-import {
-	Canvas,
-	ContextMenu,
-	TAB_ID,
-	TldrawEditor,
-	TldrawEditorConfig,
-	TldrawUi,
-} from '@tldraw/tldraw'
+import { Canvas, ContextMenu, TAB_ID, TldrawEditor, TldrawUi, createTLStore } from '@tldraw/tldraw'
 import '@tldraw/tldraw/editor.css'
 import '@tldraw/tldraw/ui.css'
 import { throttle } from '@tldraw/utils'
-import { useEffect, useState } from 'react'
+import { useLayoutEffect, useState } from 'react'
 
 const PERSISTENCE_KEY = 'example-3'
-const config = new TldrawEditorConfig()
-const instanceId = TAB_ID
-const store = config.createStore({ instanceId })
 
 export default function PersistenceExample() {
-	const [state, setState] = useState<
-		| {
-				name: 'loading'
-		  }
-		| {
-				name: 'ready'
-		  }
-		| {
-				name: 'error'
-				error: string
-		  }
-	>({ name: 'loading', error: undefined })
+	const [store] = useState(() => createTLStore({ instanceId: TAB_ID }))
+	const [syncedStore, setSyncedStore] = useState<
+		{ status: 'loading' } | { status: 'ready' } | { status: 'error'; error: string }
+	>({
+		status: 'loading',
+	})
```

Comment:
I went back and forth on these names a bunch of times. There's the "store" (`TLStore`) and then there's the "store with status and maybe error", and I really don't know what to call that. It was called `SyncedStore` before so that's what I left it. Loading state is good though.

Reply:
christ i'll just call it store with status

Reply:
:rubberduck: